### ISSUE TITLE
[Mocknet] Error while connecting burnchain db in peer loop

### ISSUE THREAD

**Issue by zone117x** [blockstack/stacks-blockchain#1455]
**Steps To Reproduce**
Run `stacks-testnet /app/config/Stacks.toml` with `Stacks.toml`:
```toml
[node]
name = "helium-node"
rpc_bind = "0.0.0.0:9000"

[burnchain]
chain = "bitcoin"
mode = "mocknet"
commit_anchor_block_within = 5000

[[mstx_balance]]
# Private key: b8d99fd45da58038d630d9855d3ca2466e8e0f89d3894c4724f0efc9ff4b51f001
address = "ST2ZRX0K27GW0SP3GJCEMHD95TQGJMKB7G9Y0X1MH"
amount = 10000000000000000

[[mstx_balance]]
# Private key: 3a4e84abb8abe0c1ba37cef4b604e73c82b1fe8d99015cb36b029a65099d373601
address = "ST26FVX16539KKXZKJN098Q08HRX3XBAP541MFS0P"
amount = 10000000000000000

[[mstx_balance]]
# Private key: 052cc5b8f25b1e44a65329244066f76c8057accd5316c889f476d0ea0329632c01
address = "ST3CECAKJ4BH08JYY7W53MC81BYDT4YDA5M7S5F53" 
amount = 10000000000000000

[[mstx_balance]]
# Private key: 9aef533e754663a453984b69d36f109be817e9940519cc84979419e2be00864801
address = "ST31HHVBKYCYQQJ5AQ25ZHA6W2A548ZADDQ6S16GP" 
amount = 10000000000000000
```

Console output (this pattern repeats):
```
stacks-blockchain_1  | Stacks block #1 (46875b8f9bc7a4ff1a357b6a16493b478d84593989268d4cfe670f0c9da92ac1) successfully produced, including 1 transactions
stacks-blockchain_1  | INFO [1587467058.817] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/tenure.rs:85] Node starting new tenure with VRF 1b2055f871a4ccde0ab743e4a1e6f15a2cfb02ed1c3d21d4b1392f6ccf6df78d
stacks-blockchain_1  | WARN [1587467059.636] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467060.637] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467061.639] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467062.641] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467063.641] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | Burnchain block #3 (12771355e46cd47c71ed1721fd5319b383cca3a1f9fce3aa1c8cd3bd37af20d7) was produced with sortition #70ec9335917332add9e6b24c26611746a20e772f9ccf6b36c3949a5ad97f51cb
stacks-blockchain_1  | ERROR [1587467063.932] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/src/chainstate/stacks/index/storage.rs:878] Not found (no file is open)
stacks-blockchain_1  | INFO [1587467063.990] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/event_dispatcher.rs:25] Connected to event observer at: 192.168.65.2:3700
stacks-blockchain_1  | Stacks block #2 (686c9ddfe49d2f896f9fa26da1245100e07be51c79ccbd89b984264458c0b4b3) successfully produced, including 1 transactions
stacks-blockchain_1  | -> Tx issued by 8bc5147525b8f477f0bc4522a88c8339b2494db5 (fee: 0, nonce: 1)
stacks-blockchain_1  |    Coinbase
stacks-blockchain_1  | INFO [1587467064.003] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/tenure.rs:85] Node starting new tenure with VRF 661cac5f0deb317dcc6f3e2a5d7759f3d8c69dbd4e134084708ae8fafa8e7d20
stacks-blockchain_1  | WARN [1587467064.643] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467065.644] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467066.645] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467067.645] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | WARN [1587467068.647] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:76] Error while connecting burnchain db in peer loop: unable to open database file
stacks-blockchain_1  | Burnchain block #4 (fe15c0d3ebe314fad720a08b839a004c2e6386f5aecc19ec74807d1920cb6aeb) was produced with sortition #901113585049a7c1a41e9584b4ea761f7dea9361c9f9b7bf6d6ea5184eb029eb
stacks-blockchain_1  | INFO [1587467069.192] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/event_dispatcher.rs:25] Connected to event observer at: 192.168.65.2:3700
stacks-blockchain_1  | -> Tx issued by 8bc5147525b8f477f0bc4522a88c8339b2494db5 (fee: 0, nonce: 2)
stacks-blockchain_1  |    Coinbase
```

**Comment by lgalabru** [comment]
Addressed with #1456

**Comment by zone117x** [comment]
Fixed, thanks @lgalabru 💯
